src/components/Imip.vue

```diff
@@ -43,32 +48,53 @@
 
 		<EventData :event="parsedEvent" />
 
+		<div v-if="showMoreOptions" class="imip__more-options">
+			<textarea />
+		</div>
+
 		<!-- TODO: actually implement buttons (https://github.com/nextcloud/mail/issues/6803) -->
 		<!-- TODO: "More options" needs more specification -->
-		<!--
-		<div class="imip__actions">
-			<template v-if="isRequest && eventIsInFuture">
-				<Button
-					type="tertiary">
-					{{ t('mail', 'Accept') }}
-				</Button>
-				<Button type="tertiary">
-					{{ t('mail', 'Decline') }}
-				</Button>
-			</template>
-			<Actions :menu-title="t('mail', 'More options')" />
+		<div
+			v-if="isRequest && !wasProcessed && eventIsInFuture && userAsAttendee"
```

Comment:
we should leave the isProcessed out - and show the option the user clicked before (if that is technically possible). Updating an event later again to change the partstat again should be possible.

Reply:
This has been discussed in a call.

## Summary

There are 2 states:
1. Event is not yet in calendar => Show buttons
2. Event is in calendar => Hide buttons, fetch event and try to display participation status

Reply:
Would it be some kind of middle ground to consider the message processed if `\answered` is set on the message? This could be *good enough* for most cases.

The ideal case is to show the latest version of the event fetched from the own calendar, of course.